### DISCUSSION TITLE
NPM RUN DEV FAILS

### DISCUSSION THREAD

**Discussion by ernst86913** [directus/directus#5336] [Archive]
VERSION:
LATEST DEV Version with latest Commits 

NPM RUN DEV FAILS with following Error:

> lerna run dev --stream --parallel

C:\Users\rer\Documents\Coding\directus\node_modules\@lerna\command\index.js:74
            writeLogFile(this.project.rootPath);
                                      ^

TypeError: Cannot read property 'rootPath' of undefined

**Comment by rijkvanzanten** [comment]
Can you try reinstalling the node modules?
